toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1369,6 +1424,25 @@ PreferencesPopup::PreferencesPopup()
                                  Qt::AlignLeft | Qt::AlignVCenter);
       generalFrameLay->addWidget(sceneNumberingCB, 0,
                                  Qt::AlignLeft | Qt::AlignVCenter);
+      QGridLayout *projectRootLay = new QGridLayout();
+      projectRootLay->setMargin(0);
+      projectRootLay->setHorizontalSpacing(5);
+      projectRootLay->setVerticalSpacing(10);
+      {
+        projectRootLay->addWidget(new QLabel(" "), 0, 0);
+        projectRootLay->addWidget(
+            new QLabel(tr("Additional Project Locations:"), this), 1, 0,
+            Qt::AlignRight | Qt::AlignVCenter);
+        projectRootLay->addWidget(m_projectRootStuff, 2, 0);
+        projectRootLay->addWidget(m_projectRootDocuments, 3, 0);
+        projectRootLay->addWidget(m_projectRootDesktop, 4, 0);
+        projectRootLay->addWidget(m_projectRootCustom, 5, 0);
+        projectRootLay->addWidget(m_customProjectRootLabel, 6, 0,
+                                  Qt::AlignRight | Qt::AlignVCenter);
+        projectRootLay->addWidget(m_customProjectRootFileField, 6, 1, 1, 3);
+        projectRootLay->addWidget(m_projectRootDirections, 7, 0, 1, 4);
+      }
+      generalFrameLay->addLayout(projectRootLay, 0);
```

Comment:
It might be a matter of taste, but how about to put these checkboxes in QGroupBox?
Like this:

<img src ="https://cloud.githubusercontent.com/assets/17974955/18537550/507962b6-7b41-11e6-877b-485feb2b9022.png" width=500>